### DISCUSSION TITLE
fargate-scheduler  Pod not supported on Fargate: invalid SecurityContext fields: Capabilities added: NET_ADMIN

### DISCUSSION THREAD

**Discussion by john-jun** [linkerd/linkerd2#7303] [Show and tell] [answered]
fargate-scheduler  Pod not supported on Fargate: invalid SecurityContext fields: Capabilities added: NET_ADMIN

**Comment by olix0r** [comment, marked as answer]
This is currently a limitation of Fargate. See https://github.com/aws/containers-roadmap/issues/682
